entry_types/scrolled/package/src/frontend/index.js

```diff
@@ -25,6 +27,7 @@ export * from './i18n';
 export * from './SectionThumbnail';
 export {default as Entry} from './Entry';
 export {EntryStateProvider} from '../entryState/EntryStateProvider'
+export {useEditorSelection} from './EditorState';
```

Comment:
Why is this needed here? Unused?

Reply:
We need to export this here to be able to use it in BeforeAfter.js